src/routes/docs/components/drawer.md

```
@@ -502,93 +452,37 @@ Use `fixed|aboslute` prop to manage the CSS position value of the drawer.
-<script>
-  import { Drawer, Button, CloseButton } from "flowbite-svelte";
-  import { InfoCircleSolid } from "flowbite-svelte-icons";
-  let hidden10 = $state(true);
-  let activateClickOutside = false;
-  let backdrop = false;
-</script>
 
-<div class="text-center">
-  <Button onclick={() => (hidden10 = false)}>Show drawer</Button>
-</div>
+### This section is useless. We don't need it. Confirm.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)